static/app/components/replays/actionCategory.tsx

```diff
@@ -3,68 +3,52 @@ import styled from '@emotion/styled';
 
 import Tooltip from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
-import {
-  BreadcrumbType,
-  BreadcrumbTypeHTTP,
-  BreadcrumbTypeNavigation,
-  RawCrumb,
-} from 'sentry/types/breadcrumbs';
+import {BreadcrumbType, RawCrumb} from 'sentry/types/breadcrumbs';
 
 import {convertCrumbType} from '../events/interfaces/breadcrumbs/utils';
 
 type Props = {
-  category: RawCrumb;
-  description?: BreadcrumbTypeNavigation | BreadcrumbTypeHTTP;
+  action: RawCrumb;
 };
 
-function getActionCategoryDescription(crumb: RawCrumb): string {
+type actionCategoryInfo = {
+  description?: string;
+  title?: string;
+};
+
+function getActionCategoryInfo(crumb: RawCrumb): actionCategoryInfo {
   switch (crumb.type) {
     case BreadcrumbType.USER:
     case BreadcrumbType.UI:
-      return t('UI Click');
-    case BreadcrumbType.HTTP:
-      return t('Fetch');
+      return {
+        title: t('UI Click'),
+        description: `${crumb.category}: ${crumb.message}`,
+      };
     case BreadcrumbType.NAVIGATION:
-      return t('Navigation');
+      return {
+        title: t('Navigation'),
+        description: `${crumb.category}: ${crumb.data?.from} => ${crumb.data?.to}`,
+      };
     case BreadcrumbType.ERROR:
-      return t('Error');
+      return {title: t('Error'), description: `${crumb.category}: ${crumb.message}`};
     default:
-      return '';
+      return {
+        title: '',
+        description: '',
+      };
```

Comment:
Will we see an empty row in this `default` case?

Reply:
in this case, we will not have any tooltip or info to display, because we are validating if the description has content or not in the template.

Reply:
It looks to me like there will be an empty row with an empty `<Value>` component (because `text` is the empty string) and then the tooltip for that will be disabled.

Reply:
oh, you are right in that case, but am thinking we never we will see an empty row, because we are contemplating some specific breadcrumb types, although I'm adding the `t('Default') in those cases. Is that Ok?

Reply:
Seems fine for now. This is a case we should notice and handle as part of the first version of the PR